packages/central-server/src/apiV2/userAccounts/RegisterUserAccounts.js

```diff
@@ -92,15 +93,25 @@ export class RegisterUserAccounts extends CreateUserAccounts {
       position,
       contactNumber,
       password,
+      primaryPlatform,
     };
 
     // Get one of the non-default platform configs if it exists
     const email = this.req?.apiClientUser?.email;
     const platformConfig = email in PLATFORM_CONFIGS ? PLATFORM_CONFIGS[email] : null;
```

Comment:
Given that we're now able to pass in the primary platform, using the apiClientEmail to work out which platform they're on feels a little unnecessary.

We still would want to retain having some custom fields populated based on the platform (`countryName` and `permissionGroupName` for LESMIS), but we could just use the `primaryPlatform` to work that out.

Thoughts on re-working this, and switching the `useRegisterUser` mutation in `@tupaia/lesmis` to pass in the `primaryPlatform`?